content/applications/finance/accounting/reporting.rst

```diff
@@ -19,136 +18,135 @@ countries :
 -  **Aged Receivable**
 -  **Cash Flow Statement**
 -  **Tax Report**
--  **Bank Reconciliation**
 
-You can annotate every reports to print them and report to your adviser.
-Export to xls to manage extra analysis. Drill down in the reports to see
-more details (payments, invoices, journal items, etc.).
+Reports can be **annotated, printed, exported to XLSX or PDF**, and **drilled down** to see details
+(payments, invoices, journal items, etc.).
 
-You can also compare values with another period. Choose how many periods
-you want to compare the chosen time period with. You can choose up to 12
-periods back from the date of the report if you don't want to use the
-default **Previous 1 Period** option.
+.. image:: reporting/reporting-annotate.png
```

Comment:
It is necessary to click the down arrow to be able to annotate, it might be good to clarify it? The image doesn't show how to print or export. How is it possible to do it?